### PR TITLE
yext css

### PR DESCRIPTION
**Pull request by michaelwoon**
I changed the markdown and css to follow the Yext CSS style, let me know if anything needs to be changed.

### CODE REVIEW

File: src/templates/media.soy [hunk 1]
```diff
@@ -34,11 +34,11 @@
 {template .booklist_item}
 {call common.layout}
 {param pageContent}
-<div class="wrapper">
-  <div class="images">
+<div class="Wrapper">
+  <div class="Wrapper-images">
     <img src="{$coverImageUrl}"?:"http://bit.ly/1B1bPGC">
```
Comment (bhainesva) [inline src/templates/media.soy]: Can you double check that this works how you expect?

Comment (michaelwoon) [reply src/templates/media.soy]: It seems to be fine when I run it, is there a different way to do it?

Comment (bhainesva) [reply src/templates/media.soy]: The `?:` stuff isn't inside curly brackets. I think it's just going to be printed into the html. 

So if $coverImageUrl was 'http://www.google.com' you would get: 

``

Comment (michaelwoon) [reply src/templates/media.soy]: Thanks for catching that, I updated mine to have
    
It works now

**Pull request merged**